website/docs/r/codeartifact_repository.html.markdown

```diff
@@ -88,8 +88,9 @@ The following arguments are supported:
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The Name of the repository.
+* `id` - The ARN of the repository.
 * `arn` - The ARN of the repository.
+* `repository` - The Name of the repository.
```

Comment:
This is defined above as an argument, so doesn't need to be added to the attributes section.

```suggestion
```